scss/civicrm/contact/pages/_reports.scss

```diff
@@ -21,12 +21,26 @@ body[class*='page-civicrm-report-'] {
     }
 
     .crm-report-form-block {
+      .report-layout.statistics-table {
+        margin-bottom: 20px;
+
+        th + td {
+          background-color: $gray-lighter !important;
```

Comment:
isn't this something that should be done for every table?

Reply:
No there can be footers with this scenario which should not be touched.
![image](https://user-images.githubusercontent.com/26058635/37203297-3bd2ab8a-23b3-11e8-8f43-a0502640d5b1.png)